### PR TITLE
Add installation processes

### PR DESCRIPTION
**Pull request by leifmadsen**
Provides the initial work for adding installation. I expect there to be a fair amount of changes required to get this ready for merge.

### CODE REVIEW

**Comment by leifmadsen** [comment]
[Update documentation to account for installation process change](https://trello.com/c/AzFRF4W7/735-update-documentation-to-account-for-installation-process-change)

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 1]
```diff
@@ -25,27 +25,249 @@
...
+. Subscribe to the AMQ Certificate Manager Operator. For more information, see <<subscribing-to-the-amq-certificate-manager-operator>>.
 . Subscribe to the Elastic Cloud on Kubernetes Operator. For more information, see <<subscribing-to-elastic-cloud-on-kubernetes-operator>>.
 . Subscribe to the Service Telemetry Operator. For more information, see <<subscribing-to-the-service-telemetry-operator>>.
 . Create a ServiceTelemetry object in {OpenShiftShort}. For more information, see <<creating-a-servicetelemetry-object-in-openshift>>.
 
-[id='enabling-the-upstream-community-catalog']
-== Enabling the upstream Community Catalog
-
 [id='creating-a-namespace']
 == Creating a namespace
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]:
```suggestion
To create a namespace, run the following command: 
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 2]
```diff
@@ -25,27 +25,249 @@
...
-== Subscribing to the {MessageBus} Operator
+[source,bash]
+----
+oc new-project service-telemetry
+----
+
+[id='enabling-the-operatorhubio-community-catalog-source']
+== Enabling the OperatorHub.io Community Catalog Source
+
+Installation of ElasticSearch is done via the Elastic Cloud on Kubernetes Operator, which must be subscribed to from the OperatorHub.io catalog source.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]:
```suggestion
Install ElasticSearch through Elastic Cloud on Kubernetes Operator. You must be subscribed to Kubernetes Operator from the OperatorHub.io catalog source.
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 3]
```diff
@@ -25,27 +25,249 @@
...
+  image: quay.io/operator-framework/upstream-community-operators:latest
+  displayName: OperatorHub.io Operators
+  publisher: OperatorHub.io
+EOF
+----
+
+[id='enabling-the-infrawatch-operator-source']
+== Enabling InfraWatch Operator Source
+
+To install {ProjectShort} we need to install an OperatorSource that contains the Service Telemetry Operator and the Smart Gateway Operator.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]: Should this be "Before you deploy STF on OpenShift, install an an OperatorSource that contains the Service Telemetry Operator and the Smart Gateway Operator."

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 4]
```diff
@@ -25,27 +25,249 @@
...
+  authorizationToken: {}
+  displayName: InfraWatch Operators
+  endpoint: https://quay.io/cnr
+  publisher: InfraWatch
+  registryNamespace: infrawatch
+  type: appregistry
+EOF
+----
+
+Validate the creation of your OperatorSource by running `oc get operatorsources`. A successful import will result in the `MESSAGE` field returning a result of `The object has been successfully reconclied`.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]: Thank you for including validation steps!

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 5]
```diff
@@ -25,27 +25,249 @@
...
+$ oc get packagemanifests | grep InfraWatch
+
+servicetelemetry-operator                    InfraWatch Operators       7m20s
+smartgateway-operator                        InfraWatch Operators       7m20s
+----
+
+[id='subscribing-to-the-amq-certificate-manager-operator']
+== Subscribing to the AMQ Certificate Manager Operator
+
+Install the `Subscription` for AMQ Certificate Manager. After creating the `Subscription` you can validate your AMQ7 Certificate Manager is available by running `oc get csv` and looking for the `amq7-cert-manager.v1.0.0` to have a phase of `Succeeded`.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]:
```suggestion
To subscribe to the AMQ Certificate Manager Operator, create the subscription and validate the AMQ7 Certificate Manager: 
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 6]
```diff
@@ -25,27 +25,249 @@
...
+  channel: alpha
+  installPlanApproval: Automatic
+  name: amq7-cert-manager
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: amq7-cert-manager.v1.0.0
+EOF
+----
+
+Validate your `ClusterServiceVersion` succeeded by running `oc get csv`.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]: To validate your `ClusterServiceVersion`, run the `oc get csv` command. Ensure that `amq7-cert-manager.v1.0.0` has a phase `Succeeded`.

Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]:
```suggestion
To validate your ClusterServiceVersion, run the oc get csv command. Ensure that amq7-cert-manager.v1.0.0 has a phase Succeeded.
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 7]
```diff
@@ -25,27 +25,249 @@
...
+$ oc get csv
+
+NAME                       DISPLAY                                         VERSION   REPLACES                   PHASE
+elastic-cloud-eck.v1.0.1   Elastic Cloud on Kubernetes                     1.0.1     elastic-cloud-eck.v1.0.0   Succeeded
+----
+
 [id='subscribing-to-the-service-telemetry-operator']
 == Subscribing to the Service Telemetry Operator
 
+Installation of the Service Telemetry Framework is done by creating a Subscription to the Service Telemetry Operator. The Service Telemetry Operator has a list of dependency Operators that will be automatically subscribed to when the Service Telemetry Operator subscription is created.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]:
```suggestion
Create a subscription to the Service Telemetry Operator. The Service Telemetry Operator automatically subscribes to a list of dependency Operators when you create the Service Telemetry Operator subscription.
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 8]
```diff
@@ -25,27 +25,249 @@
...
+  name: stf-default
+  namespace: service-telemetry
+spec:
+  eventsEnabled: true
+  metricsEnabled: true
+  highAvailabilityEnabled: false
+EOF
+----
+
+Logs for the deployment of Service Telemetry Framework can be viewed in the Service Telemetry Operator using `oc logs`.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]:
```suggestion
To view the STF deployment logs in the Service Telemetry Operator, run the `oc logs` command:
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 9]
```diff
@@ -25,27 +25,249 @@
...
+
+[source,bash]
+----
+oc logs $(oc get pod --selector='name=service-telemetry-operator' -oname) -c ansible
+----
+
+Near the end of the log you should see a playbook recap that results in no failed or changed items, which signals the operator was able to complete tasks successfully.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]:
```suggestion
If the Operator completed the tasks successfully,  the playbook summary near the end of the log file shows no failed or changed items.
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 10]
```diff
@@ -25,27 +25,249 @@
...
+
+[source,bash]
+----
+PLAY RECAP *********************************************************************
+localhost                  : ok=37   changed=0    unreachable=0    failed=0    skipped=1    rescued=0    ignored=0
+----
+
+Also look at the pods and their status to determine that all workloads are operating nominally.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]:
```suggestion
View the pods and the status of the pods to determine that all workloads are operating nominally.
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc (hunk 4 above)
Comment (mrunge) [reply doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]: Maybe better use `oc get operatorsources -n openshift-marketplace`

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 11]
```diff
@@ -25,27 +25,249 @@
...
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: servicetelemetry-operator
+  source: infrawatch-operators
+  sourceNamespace: openshift-marketplace
+EOF
+----
+
+Validation of the Service Telemetry Operator and the dependent operators can be done with `oc get csv` in the `service-telemetry` namespace.
```
Comment (mrunge) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]:
```suggestion
Validation of the Service Telemetry Operator and the dependent operators can be done with `oc get csv -n service-telemetry`.
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc [hunk 12]
```diff
@@ -25,27 +25,249 @@
...
+elastic-cloud-eck.v1.0.1            Elastic Cloud on Kubernetes                     1.0.1     elastic-cloud-eck.v1.0.0            Succeeded
+prometheusoperator.0.37.0           Prometheus Operator                             0.37.0    prometheusoperator.0.32.0           Succeeded
+service-telemetry-operator.v1.0.1   Service Telemetry Operator                      1.0.1     service-telemetry-operator.v1.0.0   Succeeded
+smart-gateway-operator.v1.0.1       Smart Gateway Operator                          1.0.1     smart-gateway-operator.v1.0.0       Succeeded
+----
+
 [id='creating-a-servicetelemetry-object-in-openshift']
 == Creating a ServiceTelemetry object in {OpenShiftShort}
+
+Create an instance of ServiceTelemetry in {OpenShiftShort} to deploy Service Telemetry Framework.
```
Comment (mrunge) [inline doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]:
```suggestion
Create an instance of ServiceTelemetry in {OpenShiftShort} to deploy the Service Telemetry Framework.
```

File: doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc (hunk 4 above)
Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_deploying-stf-to-the-openshift-environment.adoc]: Yea the intent here is to show the general command that will get used, with the output section showing the full expected command to be run.

**Comment by leifmadsen** [comment]
Changes made per review

File: doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc [hunk 13]
```diff
@@ -23,15 +23,17 @@
 [id='prerequisites-for-stf-deployment_{context}']
 = Prerequisites for {ProjectShort} installation
 
-Complete the following prerequisite tasks:
+Ensure that OpenShift version 4.x is running somewhere. For further details on how to deploy OpenShift, see the product documentation. Prior to installation of STF you need to do the following
```
Comment (mrunge) [inline doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc]:
```suggestion
Ensure that {OpenShift} version 4.x is running somewhere. For further details on how to deploy {OpenShift}, see the product documentation. Prior to installation of {ProjectShort} you need to do the following
```

File: doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc [hunk 14]
```diff
@@ -23,15 +23,17 @@
...
 
 . Deploy {OpenShift}. For more information, see https://access.redhat.com/documentation/en-us/openshift_container_platform/4.3/
 . Deploy {ProjectShort} to the {OpenShiftShort} environment. For more information, see <<deploying-stf-to-the-openshift-environment_installing-the-core-components-of-stf>>.
 . Configure {OpenStack} templates and deploy changes into the cloud infrastructure. For more information, see Completing the STF configuration.
-//. Configure OpenStack templates and deploy changes into the cloud infrastructure. For more information, see <<completing-the-stf-configuration_completing-the-stf-configuration>>.
 
+The following STF core components are managed by Operators:
```
Comment (mrunge) [inline doc-Service-Telemetry-Framework/modules/proc_prerequisites-for-stf-deployment.adoc]:
```suggestion
The following {ProjectShort} core components are managed by Operators: 
```

**Pull request merged**